assets/dev/js/editor/elements/views/base.js

```diff
@@ -659,6 +659,31 @@ BaseElementView = BaseContainer.extend( {
 		this.renderHTML();
 	},
 
+	isAtomicDynamic( dataBinding ) {
+		return !! ( dataBinding.el.hasAttribute( 'data-binding-dynamic' ) && elementorCommon.config.experimentalFeatures.e_nested_atomic_repeaters );
+	},
+
+	isTitle( bindingSetting ) {
+		const titleSettings = [ 'item_title', 'tab_title' ];
+
+		return titleSettings.includes( bindingSetting );
+	},
+
+	getDynamicValue( settings, bindingSetting ) {
+		const valueToParse = settings.attributes?.__dynamic__?.[ bindingSetting ],
+			dynamicSettings = { active: true };
+
+		if ( valueToParse ) {
+			try {
+				return elementor.dynamicTags.parseTagsText( valueToParse, dynamicSettings, elementor.dynamicTags.getTagDataContent );
+			} catch {
+				return false;
+			}
+		}
+
+		return settings.attributes[ bindingSetting ];
+	},
+
```

Comment:
Can you please update the comment here:

https://github.com/elementor/elementor/blob/main/assets/dev/js/editor/elements/views/base.js#L681

```
	 * Current Limitation:
	 * Not working with dynamics, will required full re-render.
	 * UPDATE: Support for dynamics has experimentally been added in v3.23
```

Reply:
Fixed.
Thanks :)